doc/postgres.md

```diff
@@ -0,0 +1,283 @@
+## PostgreSQL formats
```

Comment:
I think it's ok to add this in doc/ for now. Generally documentation for formats have been put in format/ (and doc/formats.md is generated from them) but current they are .jq files. They used to be .md and i'm working on making them .md again as jq was just to messy for documentation. Once that is done we can move/split this file.

Reply:
Now master uses markdown for format documentation. If you now include a file named `<format-name>.md` using `interp.RegisterFS(...)`, see mp4.go for example, and it will be used by the cli as help (`fq -h <format>` and `help(<format>` in the REPL)) and also for `make doc` to generate `doc/formats.md`.

Would it be possible to split/move this file into `format/postgres/{pg_heap,pg_control,pg_wal,pg_btree}.md` somehow? one question is how to include command example output? just include the command or maybe somehow limit the width of the output to make it more cli friendly? (you can use `-o line_bytes=10` to force less width).